Makefile

```diff
@@ -14,7 +14,7 @@ NAME = Talos
 
 ARTIFACTS := _out
 TOOLS ?= ghcr.io/siderolabs/tools:v1.3.0
-PKGS ?= v1.3.0
+PKGS ?= v1.3.0-1-g17ea5e6
```

Comment:
we should probably cut `v1.4.0-alpha.0` to avoid confusion

Reply:
fixed